src/CSSList/CSSList.php

```diff
@@ -444,17 +443,15 @@ protected function renderListContents(OutputFormat $outputFormat): string
 
     /**
      * Return true if the list can not be further outdented. Only important when rendering.
-     *
-     * @return bool
      */
-    abstract public function isRootList();
+    abstract public function isRootList(): bool;
 
     /**
      * Returns the stored items.
      *
-     * @return array<int, RuleSet|Import|Charset|CSSList>
+     * @return array<int<0, max>, RuleSet|Import|Charset|CSSList>
```

Comment:
Can it be a `list`?

Reply:
No, as `$this->contents` may have gaps in the keys due to the `unset()` in `::remove()`.